Modules/Tokens/test/Tokens.Domain.Tests/Tests/TokenTests.cs

```diff
@@ -27,4 +28,64 @@ public void Raises_TemplateCreatedDomainEvent_on_creation()
         domainEvent.TokenId.Should().Be(token.Id);
         domainEvent.CreatedBy.Should().Be(address);
     }
+
+    [Fact]
+    public void Expression_CanBeCollectedBy_null_forIdentity()
+    {
+        // Arrange
+        var token = new Token("createdByIdentityAddress", DeviceId.Parse("DVC1"), [], DateTime.Now.AddDays(1), null);
```

Comment:
Don't use the implicit conversion for the address please.

Reply:
changes made